recipes/h5glance/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "h5glance" %}
+{% set org = "European-XFEL" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dd848a967112f0a0d1d04a1739afbe7dc7bac682b93b8017e5d5a11ca8260c5d
```

Comment:
IIRC, hashes of Github archives can change. The packages I'm familiar with use the sdist (.tar.gz file) on PyPI as a starting point, but I don't know what's common across conda-forge.

Reply:
I think that that is true. I do use it this way on many other occasions